src/cloud-element-templates/Condition.js

```diff
@@ -90,3 +90,10 @@ function isPropertyAllowed(element, property) {
 
   return true;
 }
+
+
+// helpers //////////////////////
+
+function hasProperty(obj, key) {
```

Comment:
We could use `min-dash` [`has`](https://github.com/bpmn-io/min-dash/blob/main/lib/lang.js#L72) here to not re-invent the wheel.